pyvista/plotting/tools.py

```diff
@@ -459,23 +460,78 @@ def opacity_transfer_function(mapping, n_colors, interpolate=True,
 
 
 def parse_color(color, opacity=None, default_color=None):
-    """Parse color into a vtk friendly rgb list.
+    """Parse color into a VTK friendly RGB(A) list.
+
+    If ``color`` is a sequence of RGBA floats, the ``opacity`` parameter
+    is ignored.
 
     Values returned will be between 0 and 1.
 
+    Parameters
+    ----------
+    color : str or sequence
+        Either a string, RGB sequence, RGBA sequence, or hex color string.
+        For example:
+
+            * ``'white'``
+            * ``'w'``
+            * ``[1, 1, 1]``
+            * ``[0.5, 1, 0.7, 1]``
+            * ``'#FFFFFF'``
```

Comment:
The implementation seems to assume float values between 0 and 1. We should probably explicitly document this, because users might expect uint8 values to be handled correctly.

Reply:
Yea, I was actually thinking of creating a proper `Color` class with all necessary `from_*` and `to_*` conversion functions, but that would be too much out of scope for this PR. So for now I will just document it indeed.